src/utils/getBuiltinComment.ts

```diff
@@ -0,0 +1,23 @@
+const START_COMMENT = "@Preview{";
```

Comment:
The naming should be better, 
const BEG_OF_FILE_COMMENT_PATTERN instead of START_COMMENT